### PR TITLE
Create command for assigning stickers for existing paid WS leases

### PR DESCRIPTION
**Pull request by santtul**
## Description :sparkles:
- Create command for assigning stickers for existing paid WS leases

### Closes :no_good_woman:
**[VEN-940](https://helsinkisolutionoffice.atlassian.net/browse/VEN-940)** 

## Testing :alembic:
    ./manage.py assign_ws_lease_stickers

### CODE REVIEW

File: .prod/on_deploy.sh [hunk 1]
```diff
@@ -2,3 +2,5 @@
 # Runs migration scripts on each deployment
 exec python /app/manage.py migrate --noinput
 
+# This can be removed once executed
+exec python /app/manage.py assign_ws_lease_stickers
```
Comment (tebbon) [inline .prod/on_deploy.sh:6]: This won't ever be ran. The exec on line 3 will take over the shell, and the return value of that will be the whole return value of this script. If multiple commands need to be ran, then remove all execs.

**Review by tebbon — CHANGES_REQUESTED** [review]

File: .prod/on_deploy.sh (hunk 1 above)
Comment (santtul) [reply .prod/on_deploy.sh:6]: Good point! will fix it.

**Review by tebbon — APPROVED** [review]
nice!

File: leases/management/commands/assign_ws_lease_stickers.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+class Command(BaseCommand):
+
+    help = "Assign stickers for existing paid WS leases"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Assigning stickers for existing paid WS unmarked leases")
+
+        paid_ws_leases = WinterStorageLease.objects.filter(
+            status=LeaseStatus.PAID, sticker_number=None
+        )
```
Comment (a-rmz) [inline leases/management/commands/assign_ws_lease_stickers.py:18]: You can add the filter for unmarked applications here
```suggestion
        paid_ws_leases = WinterStorageLease.objects.filter(
            application__area_type=ApplicationAreaType.UNMARKED,
            status=LeaseStatus.PAID,
            sticker_number=None
        )
```

**Review by a-rmz — CHANGES_REQUESTED** [review]

**Review by a-rmz — APPROVED** [review]

**Pull request merged**